Sources/SwiftSyntax/Documentation.docc/Documentation.md

```diff
@@ -0,0 +1,8 @@
+# ``SwiftSyntax``
+
+SwiftSyntax is a set of Swift bindings for the
+[libSyntax](https://github.com/apple/swift/tree/main/lib/Syntax) library. It
```

Comment:
When previewing the documentation, I get
```
swift-syntax/Sources/SwiftSyntax/Documentation.docc/Documentation.md:4:1: warning: Link in document summary will not be displayed
Summary should only contain (formatted) text. To resolve this issue, place links and images elsewhere in the document, or remove them.
```

So I think we can’t include a link here 😢